benchmarks/nel/README.md

```diff
@@ -16,18 +16,16 @@ The following commands are defined by the project. They
 can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
 Commands are only re-run if their inputs have changed.
 
-| Command | Description |
```

Comment:
None of the `benchmarks/nel/README.md` edits look like they belong in this PR?

Reply:
Yeah, it seems like I forgot to update the readme properly in a prior PR. I regenerated the readme, it should match the project structure now.

Reply:
Please merge in `v3` and move any remaining changes to `benchmarks/nel` that aren't related to the dependency installation to a separate PR.